com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/context.py

```diff
@@ -50,8 +51,29 @@ def submit(ctxtype, graph, config = None, username = None, password = None):
     Returns:
         An output stream of bytes if submitting with JUPYTER, otherwise returns None.
     """    
+    # path to python binary
+    pythonbin = sys.executable
+    pythonreal = os.path.realpath(pythonbin)
+    pythondir = os.path.dirname(pythonbin) 
+    pythonfile = os.path.basename(pythonbin)
+    pythonrealfile = os.path.basename(pythonreal)
+    pythonconfig = pythondir+"/"+pythonfile+"-config"
```

Comment:
It is not needed and left over from my testing. I will remove line 60 when I do my next set of changes.